src/jsonsync.py

```
@@ -0,0 +1,101 @@
+        return len(self.base)
+
+    def __repr__(self):
+        return repr(self.base)
+
+    def __getattr__(self, name):
+        return getattr(self.base, name)
+
+    def compile(self):
+        if type(self.base) is list:
+            return [i.compile() for i in self.base]
+        elif type(self.base) is dict:
```

Ruff (0.12.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)